mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -212,6 +216,46 @@ public void testCreateDropTable() throws TException, IOException, InterruptedExc
     }
   }
 
+  // This test exercises the catalog level 'drop-full-table-folder' setting to see if it has an effect on
+  // dropping the full table location or not.
+  @Test
+  public void testDropTableFolderOwnershipForHiveCatalog()
```

Comment:
Maybe another test case when we do not drop the dirs?

Reply:
If it is not tested somewhere else?

Reply:
This test also covers when the setting is off and we don't drop the table dir.